dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgTypes.ts

```diff
@@ -0,0 +1,6 @@
+export type TypeOfProductionSitesNames = {
+  name: string;
+  isInHouseProductionOrIsContractProcessing: string;
+  address: string;
+  listOfGoodsAndServices: [];
```

Comment:
According to the excel it should be or instead of and, this is also wrong in the data model.

Reply:
On it.

Reply:
Done

Reply:
There are no occurences of the old spelling in my version of the code. Maybe you only see occurences in the autogenerated code? Regarding the name property: The entry "List of goods and services" in the data dictionary actually seems to require names, as stated in the corresponding question.